i18n/ko.toml

```
@@ -33,169 +81,152 @@ other = "네"
+
+[main_community_explore]
+other = "커뮤니티 둘러보기"
+
+[main_contribute]
+other = "기여하기"
 
 [main_copyright_notice]
 other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
 
-# Labels for the docs portal home page.
-[docs_label_browse]
-other = "문서 둘러보기"
-
-[docs_label_contributors]
-other = "컨트리뷰터"
+[main_documentation_license]
+other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
 
-[docs_label_users]
-other = "사용자"
+[main_github_invite]
+other = "코어 쿠버네티스 코드 베이스를 살펴보는데 관심이 있으십니까?"
 
-[docs_label_i_am]
-other = "나는..."
+[main_github_view_on]
+other = "GitHub에서 보기"
 
-# layouts > blog > pager
+[main_kubernetes_features]
+other = "쿠버네티스 기능"
 
-[layouts_blog_pager_prev]
-other = "<< 이전"
+[main_kubeweekly_baseline]
+other = "최신 쿠버네티스 뉴스 수신에 관심이 있으십니까? KubeWeekly를 신청하세요."
 
-[layouts_blog_pager_next]
-other = "다음 >>"
+[main_kubernetes_past_link]
+other = "과거 뉴스레터 보기"
 
-# layouts > blog > list
+[main_kubeweekly_signup]
+other = "구독하기"
 
-[layouts_case_studies_list_tell]
-other = "당신의 이야기를 들려주세요."
+[main_page_history]
+other ="페이지 변경 이력"
 
-# layouts > docs > glossary
+[main_page_last_modified_on]
+other = "최종 수정일시"
 
-[layouts_docs_glossary_description]
-other = "이 용어집은 쿠버네티스 용어의 종합적이고 표준화된 리스트를 제공한다. 용어집은 K8s 고유의 기술 용어 뿐만 아니라, 맥락을 이해하는데 유용한 더 일반적인 용어도 포함한다. "
+[main_read_about]
+other = "읽어 보기"
 
-[layouts_docs_glossary_filter]
-other = "태그에 따라 용어 필터링"
+[main_read_more]
+other = "더 읽기"
 
-[layouts_docs_glossary_select_all]
-other = "모두 선택"
+[note]
+other = "참고:"
 
-[layouts_docs_glossary_deselect_all]
-other = "모두 선택 해제"
+[objectives_heading]
+other = "목적"
 
-[layouts_docs_glossary_aka]
-other = "또 다른 명칭"
+[options_heading]
+other = "옵션"
 
-[layouts_docs_glossary_click_details_before]
-other = "다음"
+[post_create_issue]
+other = "이슈 생성"
 
-[layouts_docs_glossary_click_details_after]
-other = "표시를 클릭하면 각 용어에 대한 더 자세한 설명을 볼 수 있다."
+[prerequisites_heading]
+other = "시작하기 전에"
 
-# layouts > docs > search
+[seealso_heading]
+other = "더 보기"
 
-[layouts_docs_search_fetching]
-other = "결과를 가져오는 중.."
+[subscribe_button]
+other = "구독"
 
-# layouts > partial > feedback
+[synopsis_heading]
+other = "시놉시스"
 
-[layouts_docs_partials_feedback_thanks]
-other = "피드백 감사합니다. 쿠버네티스 사용 방법에 대해서 구체적이고 답변 가능한 질문이 있다면, 다음 링크에서 질문하십시오."
+[thirdparty_message]
+other = """이 섹션은 쿠버네티스에 필요한 기능을 제공하는 써드파티 프로젝트와 관련 있다. 쿠버네티스 프로젝트 작성자는 써드파티 프로젝트에 책임이 없다. 이 페이지는 프로젝트를 알파벳 순으로 나열하여 <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF 웹사이트 가이드라인</a>을 따른다. 이 목록에 프로젝트를 추가하려면 변경사항을 제출하기 전에 <a href="/contribute/style/content-guide/#third-party-content">콘텐츠 가이드</a>를 읽는다."""
```

"관련 있다" -> "관련이 있다"

---

끝 부분: "읽는다" -> "읽어본다" 정도가 낫지 않을까요?

---

@ianychoi 
말씀해주신 의견 반영하고 추가 수정해보았습니다. 이렇게 번역하는 것은 어떨까요?

```suggestion
other = """이 섹션은 쿠버네티스 써드파티 프로젝트와 관련이 있다. 쿠버네티스 프로젝트 작성자는 써드파티 프로젝트에 책임이 없다. 이 페이지는 <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF 웹사이트 가이드라인</a>에 따라 프로젝트를 알파벳 순으로 나열한다. 이 목록에 프로젝트를 추가하려면 변경사항을 제출하기 전에 <a href="/contribute/style/content-guide/#third-party-content">콘텐츠 가이드</a>를 읽어본다."""
```

---

@markruler 
"쿠버네티스에 필요한 기능을 제공하는" 에 해당하는 내용은 원문에 포함되어 있는 내용이므로 임의로 제거하는 것이 적합하지 않을 것 같습니다.. ^^

"이 섹션은 쿠버네티스 써드파티 프로젝트와 관련이 있다."
로 한글화하게 되는 경우, 다소 오해의 소지가 있을 것 같습니다. (기능에 대한 써드파티 프로젝트인지, 쿠버네티스 자체의 써드파티 프로바이더 프로젝트인지)

---

@seokho-son 의견 주셔서 감사합니다. 반영해서 커밋했습니다!